configs/test_gh_config.yaml

```diff
@@ -44,12 +44,25 @@ graphhopper:
       weighting: fastest
       # sets a zero speed for a section of Baseline Road in Roseville (OSM way id 76254223)
       custom_speed_file: test-data/custom_speeds/baseline_road_zero_speed.csv # mvn tests run from the /web folder
-    - name: bike
+    - name: bike_default  # _default suffix required for comparison against Thurton Drive custom speeds profile
```

Comment:
nit: the unit tests use this for the baseline road comparison, not thurton drive. same with the `foot_default` comment

Reply:
oof, good catch -  updated!